charts/kubescape-operator/templates/node-agent/clusterrole.yaml

```diff
@@ -37,4 +37,7 @@ rules:
 - apiGroups: ["kubescape.io"]
   resources: ["operatorcommands/status"]
   verbs: ["get", "watch", "list", "update", "patch"]
+- apiGroups: ["events.k8s.io"]
+  resources: ["events"]
+  verbs: ["create", "patch", "get"]
```

Comment:
why not ["get", "watch", "list", "update", "patch"]? you don't need them?

Reply:
Why would I need them?

Reply:
I saw them in your other PR, that's why I ask